plugins/janus_videoroom.c

```diff
@@ -9231,6 +9243,7 @@ static void *janus_videoroom_handler(void *data) {
 										h264_profile = NULL;
 										ps->vcodec = videoroom->vcodec[i];
 										ps->pt = janus_videocodec_pt(ps->vcodec);
+										ps->fmtp = g_strdup(vp9_profile);
```

Comment:
This is wrong. The profile we extracted is not the content of the fmtp line.

Reply:
Before I made these changes, the `fmtp` variable was not being used for anything in this branch. A question I have is how relevant it really is. Should it just be removed and replaced with a specific variable for `h264_profile` and `vp9_profile`?